tests/interfaces/test_batch_autograd.py

```diff
@@ -167,6 +167,36 @@ def cost(a):
         spy_gradients.assert_called()
 
 
+class TestBatchTransformExecution:
+    """Tests to ensure batch transforms can be correctly executed
+    via qml.execute and map_batch_transform"""
+
+    def test_no_batch_transform(self, mocker):
+        """Test that batch transforms can be disabled"""

```

Comment:
It looks like the test ends up testing both cases (disabled/enabled), maybe worth renaming or even making it a function-based test rather than a class-based test.

Reply:
```suggestion
    def test_no_batch_transform(self, mocker):
        """Test that batch transforms can be disabled and enabled"""
```

@trbromley yeah I struggled to write tests for this without going excessive with mocking 🤔